sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -102,9 +103,16 @@ private[spark] object HiveUtils extends Logging {
 
   val HIVE_METASTORE_JARS_PATH = buildStaticConf("spark.sql.hive.metastore.jars.path")
     .doc(s"Comma separated path of Hive jars, support both local and remote paths," +
-      s"we support path wildcards such as `hdfs://path/to/jars/*`, but not support" +
-      s"nested path wildcards such as `hdfs://path/to/jars/*/*`. When ${HIVE_METASTORE_JARS}" +
-      s"is set to `path`, we will use Hive jars configured by this")
+      s"Such as:" +
+      s" 1. /path/to/jar/xxx.jar" +
+      s" 2. file:///path/to/jar/xxx.jar" +
+      s" 3. local:///path/to/jar/xxx.jar" +
+      s" 4. hdfs://path/to/jar/xxx.jar" +
```

Comment:
how about `s3://`?

Reply:
> how about `s3://`?

As https://github.com/apache/spark/pull/29881#issuecomment-701211241 mentioned. Seems S3's method need call ClassLoader and cause stackoverflow,  I'm not familiar with S3's API,  and maybe we can make a new jira and hope some expert to solve..